HW03/veaceslav_irizanov/veaceslav_irizanov_hw_03_t_03.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'time'
+
+class Homework3
+  ZERO_S = '0'
+  ZERO_I = 0
+  ONE = 1
+  TWO = 2
+  CONDITION = 'Calling core with action'
+
+  def task3(log)
```

Comment:
I will give you advice on how to create a good solution, so try refactoring your code:
- Split log
- Select the necessary lines that match the condition
- Go through them with an iterator and select the necessary data (I advise you to use a regular expression)
- Make a difference calculation method (Time.parse is handy to use)
- Create a condition if the selected data exists, then pass it to the created method for calculating the difference in events (in task3 method)